elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -378,11 +380,14 @@ export class PeerPool extends EventEmitter {
 	public triggerNewConnections(
 		peers: ReadonlyArray<P2PDiscoveredPeerInfo>,
 	): void {
-		const peersCount = this.getPeersCountByKind();
+		const disconnectPeers = peers.filter(
```

Comment:
does this mean this is peers that is not connected?
disconnectedPeers might be better choice
'